Assignment/Homework_1 - HTML/pokedex.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Pokedex!</title>
+	</head>
+	<style type="text/css">
+		header {
+			background: #BD5858;
+		}
+		h1 { 
+			color: #FFFFFF;
+		}
+		h2 {
+			background-color: #5D90CB;
+			color: #FFFFFF;
+		}
+		footer {
+			background: #5D90CB;
+			text-align: center;
+		}
+	</style>
+	<body background="assets/backgrounds/background_home.png">
+		<!-- To Do : No background image for header-->	
+		<header id="header">
+			<!-- To Do : Make these on the same line-->
+			<button>Dropdown</button>
+			<h1>Pokedex</h1>
+			<img src="assets/icons/pokeball_icon.png">
+		</header>
+		<div id="pokedex-main">
+			<div id="search-bar">
+				<form>
+					<input type="text" name="search-keyword" placeholder="Search Pokemon">
```

Comment:
great job! can try to understand the difference between using name versus id. it'll come useful later.